tools/README.md

```diff
@@ -0,0 +1,20 @@
+# How to run spelling check?
+
+`$ sh check-spelling.sh ${doc}`
+
+# How to add words to Bitmark dictionary?
+
+`en_Bitmark.dic` contains a list of words, one per line. The first line of the dictionaries contains the approximate word count (for optimal hash memory size). Each word may optionally be followed by a slash ("/") and one or more flags, which represents the word attributes, for example affixes. The supported flags are:
+
```

Comment:
I'd say "the word attributes, such as prefixes and suffixes." Affixes isn't widely used. (I had to look it up!)